.github/workflows/build.yml

```diff
@@ -163,8 +117,8 @@ jobs:
 
   push-tags:
     runs-on: ubuntu-latest
-    needs: build
-    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    needs: ['build']
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push' && ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
I think this has had an unexpected effect, as it looks like it pushed to the deployment tags from your PR and triggered a sirius deployment, which should only happen on main branch builds, not PR's